.github/workflows/upload-models.yaml

```
@@ -1264,6 +1264,32 @@ jobs:
             ls -lh $d.tar.bz2
           done
 
+      - name: Quds v4 Persian
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)